content/docs/concepts/components/buildpack.md

```diff
@@ -39,6 +39,9 @@ During build the buildpacks contribute to the final application image. This cont
 some environment variables within the image, creating a layer containing a binary (e.g: node, python, or ruby), or 
 adding app dependencies (e.g: running `npm install`, `pip install -r requirements.txt`, or `bundle install`).
 
+## Distribution
+
+Buildpacks can be **packaged** as OCI images on an image registry or Docker daemon. This includes meta-buildpacks.
```

Comment:
We should create a separate page under `Buildpack Author` that we can link to from here. Similar to content/docs/buildpack-author-guide/create-buildpack/packaging.md but isolated. Potentially using samples repo.

Reply:
Just pushed a change that adds a reference page for the package config. I started to write the step-by-step guide for a buildpack author but it felt a bit heavy-handed given that it's really just two steps (write the toml and run the pack command) and the intricacies of writing the toml are covered in the reference page. Happy to add though, if you really think it's required.